components/RichText/Quote.module.scss

```diff
@@ -2,39 +2,41 @@
     @include border-radius-m;
 
     display: flex;
-    justify-content: center;
     margin: $spacing-6 auto;
-    padding: $spacing-4 0;
-}
+    padding: $spacing-4;
+    background: #f9fafb;
```

Comment:
✖️ 
This should be extracted into a variable, or maybe even aligned with existing shade of gray. Please consult with Hugo on that one.